python/adelphi/bin/adelphi

```diff
@@ -79,9 +107,27 @@ def export_gemini(ctx):
             log.error("Gemini schema doesn't support multiple keyspaces.")
             exit(2)
 
-    my_fn = partial(cluster_fn, my_export_fn, ctx.obj['keyspace_names'], build_options(ctx))
+    my_fn = partial(cluster_fn, my_export_fn, ctx)
     with_cluster(my_fn, **({k:ctx.obj[k] for k in ["hosts","port","username","password"]}))
 
 
+@export.command()
+@click.option('--repo-user', help='Username for the repo used for schema storage on Github')
+@click.option('--repo-token', help='Access token for schema storage on Github')
+@click.option('--repo-name', help='Name of the repo used for schema storage on Github')
+@click.option('--author-name', help='Github username to be used for schema commits')
+@click.pass_context
+def github(ctx, repo_user, repo_token, repo_name, author_name):
```

Comment:
At the moment Adelphi requires us to pass in each of these args every time, which is just a little silly.  It also makes the invocation fairly large; a sample:

```
adelphi --no-anonymize --keyspaces=testkeyspace,spacecraft --output-dir=foo github --repo-user=absurdfarce --repo-token=<long repo token> --repo-name=schema-playground --author-name=grighetto
```

Click does support the ability to pull these args from environment variables (see https://click.palletsprojects.com/en/7.x/arguments/#environment-variables) and this is probably preferred for the Github-related configs since they won't change much from one run to another.